harmonic/evidence.py

```diff
@@ -247,10 +251,36 @@ def add_chains(self, chains):
         Y = chains.ln_posterior
         nchains = self.nchains
 
+        if not num_slices is None:
```

Comment:
Totally nit-picky I'd say "if num_slices is not None" is more readable.

Reply:
Fixed!